README.md

```diff
@@ -160,9 +196,32 @@ lets you plugin an entire sails project directly into a Trails Application.
 [`bootstrap`](https://github.com/trailsjs/trailpack-bootstrap) extends the Trails
 boot process so that a custom method can be run during application startup.
 
+
+```js
+const ExtensionTrailpack = require('trailpack/extension')
```

Comment:
extension, system, tool ? Did I miss something here ? :D  did they have specificities like server/datasote or it's just to categorize them ?